README.md

```diff
@@ -50,6 +50,18 @@ cd alacritty-git
 makepkg -isr
 ```
 
+### Debian/Ubuntu
+
+Using `cargo deb` you can create and install a deb file.
```

Comment:
I'm not entirely sure about this and it's pretty much completely irrelevant and a super tiny nitpick, but I believe adding a `,` after `cargo deb` might be a good choice.

(Like this: "Using `cargo deb`, you can create and install a deb *(maybe .deb?)* file.")